examples/applications/svm_gui.py

```diff
@@ -24,7 +24,12 @@
 matplotlib.use('TkAgg')
 
 from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg
-from matplotlib.backends.backend_tkagg import NavigationToolbar2TkAgg
+try:
+    from matplotlib.backends.backend_tkagg import NavigationToolbar2Tk
+except ImportError:
+    from matplotlib.backends.backend_tkagg import (
```

Comment:
Add a comment stating this is for matplotlib `2.*` support.

Reply:
@thomasjpfan Just did that! is it ready for merge?